pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -550,14 +550,15 @@ public final synchronized void readEntriesComplete(List<Entry> entries, Object c
         if (serviceConfig.isDispatcherDispatchMessagesInSubscriptionThread()) {
             // setting sendInProgress here, because sendMessagesToConsumers will be executed
             // in a separate thread, and we want to prevent more reads
+            havePendingRead = true;
```

Comment:
the comment `setting sendInProgress here` is no more true. I'm just wondering why `havePendingRead` is set to true instead of `sendInProgress`?

Reply:
Great catch @lhotari. It is supposed to be `sendInProgress`.

Reply:
Yes @michaeljmarshall , It has to be `sendInProgress`.
@lhotari  Thanks a lot!